### DISCUSSION TITLE
Incorrect DB config, or could not connect to DB PDOException: could not find driver

### DISCUSSION THREAD

**Discussion by dhuyvetter** [YOURLS/YOURLS#3665] [General]
On my existing Yourls installation I am now getting this error:

> Fatal error
> 
> Incorrect DB config, or could not connect to DB
> PDOException: could not find driver

It is version 1.9.2 on Apache2 with PHP 8.2.3. I have checked my database and I can access it with the credentials in config.php. I have checked phpinfo and the PDO driver is installed:

```
pdo_mysql

PDO Driver for MySQL => enabled
Client API version => mysqlnd 8.2.3
```

Not sure what else I can do to debug, I've set YOURLS_DEBUG to true but nothing is showing up.

Although the error is the same as #3639 in my case PDO for mySQL driver is present, so it is a different issue.

`php8.2-mysql is already the newest version (8.2.3-1+ubuntu18.04.1+deb.sury.org+1).`

**Comment by uglyeoin** [comment]
I had to install the "nd_pdo_mysql" PHP module then it worked.
